moncli/entities/column_value.py

```diff
@@ -534,64 +534,53 @@ def __init__(self, **kwargs):
     def lat(self):
         """ The latitude value"""
         try:
-             return loads(self.value)['lat']
+            return loads(self.value)['lat']
         except KeyError:
-            return None
+            return self.value
         
     @lat.setter
     def lat(self, latitude):
-        print("lat setter")
```

Comment:
None values still need to be accounted for.  We can do this by adding the following code before the latitude value check on line 543:

`
try:
    float(latitude)
except TypeError:
    if not latitude: # This is our null check
        return self.set_value(lat=latitude)
    raise LocationError("Invalid Latitude")
`